src/System.Threading/ref/System.Threading.cs

```diff
@@ -118,7 +118,7 @@ public partial class EventWaitHandle : System.Threading.WaitHandle
     }
     public sealed partial class ExecutionContext : System.IDisposable, System.Runtime.Serialization.ISerializable
     {
-        private ExecutionContext() { }
+        internal ExecutionContext() { }
```

Comment:
Are we OK with this change?

Reply:
Has the same affect so I wouldn't fight the tooling.

Reply:
Which tool is used to generate the refs? Maybe not worth filing a bug but still seems like a bug.

Reply:
Run the following from the S.Threading ref directory:

> msbuild /t:GenerateReferenceSource

Reply:
thanks!